src/components/Table/GridTable.tsx

```diff
@@ -69,8 +69,10 @@ export interface GridStyle {
 }
 
 export interface NestedCardsStyle {
-  /** The space between top-level cards. */
-  topLevelSpacerPx: number;
+  /** Space between each card. */
+  spacerPx: number;
+  /** Width of the first and last "hidden" columns used to align nested columns. */
+  firstLastColumnWidth?: number;
```

Comment:
Since calculating this was causing horizontal layout shifts and we seemed to have static numbers, we moved to this.